package.json

```diff
@@ -11,7 +11,7 @@
   "scripts": {
     "check-main": "branch=$(git rev-parse --abbrev-ref HEAD) && [ $branch = main ] || (echo 'Error: New release can only be published on main branch' && exit 1)",
     "lint": "eslint . && prettier --list-different packages/**/*.{js,json}",
-    "prettier:write": "prettier --write packages/**/*.{js,json}",
+    "prettier:write": "prettier --write **/*.{js,json}",
```

Comment:
This was missed when `cli-lib` got separated out, so the precommit hasn't actually been linting for me 😢 
Going to run it in a separate PR.